input/docs/handbook/collections/index.md

```diff
@@ -135,7 +135,9 @@ public class ViewModel : ReactiveObject
 ```cs
 public class ComplexService 
 {
-    public SourceList<bool> Items { get; } = new SourceList<bool>();
+    private readonly SourceList<bool> _items = new SourceList<bool>();
+
+    public IObservableList<bool> Items => _items;
```

Comment:
Yeah, we should remove IObservableList<bool> in this scenario. I seen so many people get confused by it and try to bind with it.

Expose

`public IObservable<IChangeSet<bool>> Connect()`

Reply:
We can just if you want to make it easier for the end user just expose with a => _items.Connect(); but worth mentioning about `Publish()` and doing `Connect()`